cp3pt0-deployment/setup_singleton.sh

```diff
@@ -134,17 +146,17 @@ function install_licensing() {
         SOURCE_NS="ibm-licensing"
```

Comment:
Could you help to update the line 139 for `ibm-licensing-operator` ?
```
is_sub_exist "ibm-licensing-operator-app" # this will catch the packagenames of all cert-manager-operators
```